Documentation/ApiOverview/Localization/LocalizationApi/LanguageService.rst

```diff
@@ -12,5 +12,27 @@ be created directly, therefore its constructor is internal. Create a
 
 In the backend context a :php:`LanguageService` is stored in the global
 variable :php:`$GLOBALS['LANG']`.
+In the frontend it can be accessed via the contentObject:
```

Comment:
Can you please use the correct wording what you mean behind `it`. While first reading I interprete that as `$GLOBALS['LANG']` which can't be as that variable does not exists in FE requests.

Reply:
This sentence is not from me. And it is meant only for the backend. 
What is wrong with it?

Reply:
Hello Franz. The green line is from your commit. It contains the word `it` and I don't know on which word/class the `it` relates to.